cli/dts/lib.deno.unstable.d.ts

```diff
@@ -502,7 +552,8 @@ declare namespace Deno {
         }
       : never;
 
-  /** A foreign function as defined by its parameter and result types.
+  /** **UNSTABLE**: New API, yet to be vetted.
+   * A foreign function as defined by its parameter and result types.
```

Comment:
Pretty sure you need an extra newline between these, otherwise they'll be processed as a single paragraph in markdown.